Data/test.lua

```diff
@@ -0,0 +1,9646 @@
+          Nature="AUTOMATIC",
+          Name="Bard ~ Bardic Countersong",
+          Conditions={
+            function (character)
+              return (character.Variables["Bard_CF_CounterSong"] == 0)
+            end,
+          },
+        },
+      },
+      WeaponBonusProficiencySelections={
+      },
+    },
+    {
+      Level="1",
+      Abilities={
+        {
+          Category="Bard Class Feature",
+          Nature="AUTOMATIC",
+          Name="Bard ~ Bardic Performance",
+          Conditions={
+            function (character)
+              return (character.Variables["Bard_CF_BardicPerformance"] == 0)
+            end,
+          },
+        },
+      },
+      WeaponBonusProficiencySelections={
+      },
+    },
+    {
+      Level="1",
+      Abilities={
+        {
+          Category="Bard Class Feature",
+          Nature="AUTOMATIC",
+          Name="Bard ~ Armored Casting",
+          Conditions={
+            function (character)
+              return (character.Variables["Bard_CF_ArmoredCasting"] == 0)
+            end,
+          },
+        },
+      },
+      WeaponBonusProficiencySelections={
+      },
+    },
+    {
+      Level="1",
+      Abilities={
+        {
+          Category="Bard Class Feature",
+          Nature="AUTOMATIC",
+          Name="Bard ~ Cantrips",
+          Conditions={
+            function (character)
+              return (character.Variables["Bard_CF_Cantrips"] == 0)
+            end,
+          },
+        },
+      },
+      WeaponBonusProficiencySelections={
+      },
+    },
+    {
+      Level="2",
+      Abilities={
+        {
+          Category="Bard Class Feature",
+          Nature="AUTOMATIC",
+          Name="Bard ~ Versatile Performance",
+          Conditions={
+            function (character)
+              return (character.Variables["Bard_CF_VersatilePerformance"] == 0)
+            end,
+          },
+        },
+      },
+      WeaponBonusProficiencySelections={
+      },
+    },
+    {
+      Level="2",
+      Abilities={
+        {
+          Category="Bard Class Feature",
+          Nature="AUTOMATIC",
+          Name="Bard ~ Well-Versed",
+          Conditions={
+            function (character)
+              return (character.Variables["Bard_CF_WellVersed"] == 0)
+            end,
+          },
+        },
+      },
+      WeaponBonusProficiencySelections={
+      },
+    },
+    {
+      Level="5",
+      Abilities={
+        {
+          Category="Bard Class Feature",
+          Nature="AUTOMATIC",
+          Name="Bard ~ Lore Master",
+          Conditions={
+            function (character)
+              return (character.Variables["Bard_CF_LoreMaster"] == 0)
+            end,
+          },
+        },
+      },
+      WeaponBonusProficiencySelections={
+      },
+    },
+    {
+      Level="10",
+      Abilities={
+        {
+          Category="Bard Class Feature",
+          Nature="AUTOMATIC",
+          Name="Bard ~ Jack of All Trades",
```

Comment:
Is what this ability does listed somewhere?  Or is this _just_ the "classes.lst" file (and all the abilities are in class_abilities.lst).  I sort of would like if it we could merge those, so abilities aren't scattered all over the place.  But maybe that's not helpful, since we don't expect to be editting these files a lot?

Reply:
`cr_abilities_class.lst` has the definitions of these abilities.